packages/blunderbuss/README.md

```diff
@@ -0,0 +1,43 @@
+# conventional-commit-lint
```

Comment:
Do we need a README for this?  If we do keep it, can we make it reflect what this does?